components/Members/MemberCard/MemberCardInfo.tsx

```diff
@@ -15,27 +33,37 @@ export const MemberCardInfo = ({
   shortDescription,
   longDescription,
   joinDate,
+  isAdmin,
 }: MemberCardInfoProps) => {
   return (
-    <div className="flex flex-col rounded-md p-1">
+    <div className="flex flex-col rounded-md p-[10px] pr-6 w-full">
       <div className="flex gap-6">
         <ImageWithFallback
-          width="48"
-          height="48"
+          width="50"
+          height="50"
           src={imgUrl}
           alt={name}
           fallback={imgUrl}
         />
 
         <div className="titles-container block">
-          <p className="text-base">{name}</p>
-          <p className="text-sm text-slate-500 dark:text-slate-300">
+          <div className="flex items-center">
+            <p className="text-base ml-4">{name}</p>
+            {isAdmin && (
+              <Crown className="w-4 h-4 fill-[#4338CA] dark:fill-[#DBC927]" />
+            )}
```

Comment:
How about we export this into a separate component? seems like something that can be used in the future across the website

Reply:
@Tamir198 
> How about we export this into a separate component? seems like something that can be used in the future across the website

Fine by me :) you can export this into a separate component